ironic-config/ipxe_config.template

```diff
@@ -0,0 +1,81 @@
+#!ipxe
```

Comment:
I know Ironic allows copying this file, but I'd much rather prefer we don't do it. You're going to miss any upstream fixes, and reconciling will be very hard. Can we add the missing functionality in Ironic?

Reply:
I would like to get this in first, but for sure I can later implement changes from the Ironic side also I agree eventually a natively supported approach should be the way to do it.